apps/meteor/server/lib/resetUserE2EKey.ts

```diff
@@ -54,6 +54,14 @@ const sendResetNotification = async function (uid: string): Promise<void> {
 	}
 };
 
+const addUserToE2EKeysWaitingQueue = async (uid: string): Promise<void> => {
+	const subscribedRoomIds = (await Subscriptions.findByUserId(uid, { projection: { rid: 1 } }).toArray()).map((sub) => sub.rid);
```

Comment:
I think we don't need to do this. Let's set the data directly to rooms that are encrypted.

Reply:
But we don't know the subscribed rooms of the user.

Using this I'm getting the list of all subscribed roomIds, then setting the data for rooms which are encrypted out of these subscribed rooms.

Is there some better way to do this 🤔